component/pom.xml

```diff
@@ -125,10 +125,10 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>2.0</version>
+                <version>3.5.1</version>
```

Comment:
do not override the version unless there is a specific reason. Always use the one inheriting from parent pom